src/error.rs

```diff
@@ -0,0 +1,35 @@
+use std::error;
+use std::fmt;
+
+#[derive(PartialOrd, PartialEq, Clone, Debug)]
```

Comment:
FlagError will not be compared to each other.

```suggestion
#[derive(PartialEq, Clone, Debug)]
```